robot/EDA/tests/browser/settings/account_model/admin_household.robot

```diff
@@ -37,4 +54,26 @@ Validate admin and household account record type settings are updated in hierarc
     ${admin_acc_rt}=                            Get Custom Settings Value       Administrative_Account_Record_Type__c
     ${hh_acc_rt}=                               Get Custom Settings Value       Household_Addresses_RecType__c
     Should Be Equal As Strings                  ${admin_acc_rt}      ${admin_id}
-    Should Be Equal As Strings                  ${hh_acc_rt}         ${hh_id}
\ No newline at end of file
+    Should Be Equal As Strings                  ${hh_acc_rt}         ${hh_id}
+
+Validate admin and household account naming format settings are updated in hierarchy settings
+    [Documentation]         Verifies the admin and household account naming settings can be updated
+    ...                     in EDA Settings page and the same are updated in hierarchy
+    ...                     settings under custom settings.
+    [tags]                                      unstable        W-9294235       rbt:high
+    Select settings from navigation pane        Account Model Settings
+    Click action button on new EDA settings     Edit
+    Scroll to field                             Administrative Account Name Format
+    Click on hub link                           Administrative Account Name Format      Tell Me More
```

Comment:
Will the "Tell Me More" string change for different settings? If it is same, you can hardcode it in the keyword. But I'll leave it to you since you know if this info is important to be kept in the robot script as well.

Reply:
Right now it doesn't change. I just kept it open for any future help link validations as new EDA settings page is evolving now